.github/workflows/microbenchmark.yml

```diff
@@ -0,0 +1,52 @@
+name: microbenchmark
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
+    tags:
+      - "v*.*.*"
+
+# limit the access of the generated GITHUB_TOKEN
+permissions:
+  contents: read
+
+env:
+  GO_VERSION: 1.15.10
+
+jobs:
+  microbenchmark:
+    runs-on: ubuntu-latest
+    # wait up to 1 hour
+    timeout-minutes: 60
+    steps:
+      - id: buildkite
+        name: Run buildkite pipeline
+        uses: elastic/apm-pipeline-library/.github/actions/buildkite@current
+        with:
+          vaultUrl: ${{ secrets.VAULT_ADDR }}
+          vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
+          vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          pipeline: apm-agent-microbenchmark
+          waitFor: true
+          printBuildLogs: true
+          buildEnvVars: |
+            script=scripts/jenkins/bench.sh
```

Comment:
**suggestion**: can we move the scripts or rename the jenkins folder to avoid confusion ?

Reply:
That works too!